src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -43,6 +55,14 @@ const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ fagsak }) =
             ) : (
                 <AlertStripe type="feil">{errorMessage}</AlertStripe>
             )}
+            {redirect && (
+                <div>
+                    <br />
```

Comment:
Try to do this with css rather than many divs and breaks. We should be able to handle this with just Knapp and some css for positioning.

Reply:
Ok, I will use the back button